view_test.go

```diff
@@ -72,18 +72,18 @@ func (v *View) Reopen() error {
 	return v.Open()
 }
 
-// MustSetBits sets bits on a row. Panic on error.
+// MustSetColumns sets columns on a row. Panic on error.
 // This function does not accept a timestamp or quantum.
-func (v *View) MustSetBits(rowID uint64, columnIDs ...uint64) {
+func (v *View) MustSetColumns(rowID uint64, columnIDs ...uint64) {
 	for _, columnID := range columnIDs {
 		if _, err := v.SetBit(rowID, columnID); err != nil {
 			panic(err)
 		}
 	}
 }
 
-// MustClearBits clears bits on a row. Panic on error.
-func (v *View) MustClearBits(rowID uint64, columnIDs ...uint64) {
+// MustClearColumns clears columns on a row. Panic on error.
```

Comment:
I don't like this name change - sounds like we're going to clear a bunch of columns vertically, but we're just clearing bits in a row.